java/code/src/com/suse/manager/hub/SyncController.java

```diff
@@ -81,13 +81,13 @@ public SyncController(HubManager hubManagerIn) {
      * Initialize the API routes
      */
     public void initRoutes() {
-        post("/iss/sync/ping", asJson(usingTokenAuthentication(this::ping)));
-        post("/iss/sync/registerHub", asJson(usingTokenAuthentication(allowingOnlyUnregistered(this::registerHub))));
-        post("/iss/sync/storeCredentials", asJson(usingTokenAuthentication(allowingOnlyHub(this::storeCredentials))));
-        get("/iss/sync/managerinfo", asJson(usingTokenAuthentication(allowingOnlyHub(this::getManagerInfo))));
-        post("/iss/sync/storeReportDbCredentials",
+        post("/hub/ping", asJson(usingTokenAuthentication(this::ping)));
+        post("/hub/sync/registerHub", asJson(usingTokenAuthentication(allowingOnlyUnregistered(this::registerHub))));
+        post("/hub/sync/storeCredentials", asJson(usingTokenAuthentication(allowingOnlyHub(this::storeCredentials))));
+        get("/hub/managerinfo", asJson(usingTokenAuthentication(allowingOnlyHub(this::getManagerInfo))));
+        post("/hub/storeReportDbCredentials",
                 asJson(usingTokenAuthentication(allowingOnlyHub(this::setReportDbCredentials))));
-        post("/iss/sync/removeReportDbCredentials",
+        post("/hub/removeReportDbCredentials",
                 asJson(usingTokenAuthentication(allowingOnlyHub(this::removeReportDbCredentials))));
```

Comment:
Maybe at some point we should create a separate `HubController` for all the methods in `/hub/...` and a keep `SyncController` only for `/hub/sync/...` methods.

Reply:
We renamed this to `HubController` now.
If needed we can later split out a `SyncController` for URLs starting with "hub/sync/".